### DISCUSSION TITLE
love this!

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#37] [General]
THIS IS AMAZINGGG
![image](https://user-images.githubusercontent.com/84712013/159340465-c9364a69-60f7-4ab5-ace2-a49a95398b39.png)

**Comment by TechStudent10** [comment]
**w    h   a   t**

Where is that?

**Reply by KendallDoesCoding** [reply]
Oh ig it only shows to me and other accounts?

Basically, it shows the starts thing and other stuff you may like in that tab, which is epic.


![image](https://user-images.githubusercontent.com/84712013/159422271-e654943f-ab4d-4f09-a722-3687062c8376.png)

**Reply by TechStudent10** [reply]
Yeah I found it. It's in open-beta so everyone has it. 

I just don't see the point. Why not just merge this with the original feed.

**Reply by KendallDoesCoding** [reply]
> I just don't see the point. Why not just merge this with the original feed.

I think they will at some point, as this is just the BETA, but I like that it's in another feed, coz I start many repos, it's not that I want to be updated with all of them in my main feed.
